content/examples/basic/front-matter/en.md

```diff
@@ -63,3 +67,11 @@ notification:
         icon: 'download'
 ```
 You can also add a site-wide notification via the config file for params (ex. `config/_default/params.toml`)
+
+tags
+: By adding tags in the front-matter, they will be displayed in both the header and footer of the content page. Tags link to `/search?q=tag:{name}` and initiate a search for all documents that contain the respective tag.
+```
+tags:
```

Comment:
Do you mean `/search?q=tag`? Because it does link to the specified tag-name?

Reply:
> Do you mean `/search?q=tag`? Because it does link to the specified tag-name?

No, check for example the docs where tags are already in use.
For instance:

tags:
    - Ecosystem
    - cue command